### ISSUE TITLE
RuntimeWarning: coroutine 'AioRequestSigner.generate_presigned_url' was never awaited

### ISSUE THREAD

**Issue by yoav-orca** [aio-libs/aiobotocore#868]
**Describe the bug**
I'm trying to upgrade from aiobotocore 0.12.0 to 1.3.0, I'm getting the following error when trying to use `copy_snapshot` with an encrypted snapshot

**Checklist**
- [X] I have reproduced in environment where `pip check` passes without errors
- [X] I have provided `pip freeze` results
- [X] I have provided sample code or detailed way to reproduce
- [X] I have tried the same code in botocore to ensure this is an aiobotocore specific issue
- [X] I have tried similar code in aiohttp to ensure this is is an aiobotocore specific issue
- [X] I have checked the latest and older versions of aiobotocore/aiohttp/python to see if this is a regression / injection

**pip freeze results**
```
aiobotocore @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/42/75/58/e462bba721498695401c779a69d2ed88f825fb97489b0f1bcdfee4cdb0/aiobotocore-1.3.0.tar.gz
aiohttp @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/ea/55/93/7731cad4d2a4eff06c090af7a9b870111a4339ad640a3b623e51cdf826/aiohttp-3.7.4.post0-cp39-cp39-macosx_10_14_x86_64.whl
aioitertools @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/db/e9/1b/2c762c711d59d1fe774b69201ef5b0d478182bac2d02195b3f95638537/aioitertools-0.7.1-py3-none-any.whl
async-timeout @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/e1/e7/dc/347dacf16e20e4b15a992103281583f3a03db422172dec1c5f08d68c07/async_timeout-3.0.1-py3-none-any.whl
attrs @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/6f/a9/ee/569c37f69a8c365ee41d2340aeac0214ee8c0086b8d8db43a21545204b/attrs-21.2.0-py2.py3-none-any.whl
botocore @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/cd/48/a3/d12db67f407cbeaa09c5d080de64681c0143d50ec94aaf963c8ffbf9d6/botocore-1.20.49-py2.py3-none-any.whl
chardet @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/11/63/f9/797eda27963177a6b75a340f62aa194d462ea69e6b0dbb77a651fa2b62/chardet-4.0.0-py2.py3-none-any.whl
idna @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/ff/0d/80/b45ae1ddba13a20e406682fd42ce6c201d9bedc5f592bee00cf799a1fb/idna-3.2-py3-none-any.whl
jmespath @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/2c/f0/52/b0ba93d941bd49c8719dee7ca27d2096bf96e17948667388c3ee2ac8f8/jmespath-0.10.0-py2.py3-none-any.whl
multidict @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/6f/f8/91/f3a9ce599ce298de994c997dd4cedc9c6bd53d1c4ce398aa34eb24e7c9/multidict-5.1.0-cp39-cp39-macosx_10_14_x86_64.whl
python-dateutil @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/93/67/cf/49f56d9e954addcfc50e5ffc9faee013c2eb00c6d77d56c6a22cb33b54/python_dateutil-2.8.1-py2.py3-none-any.whl
six @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/08/9f/47/c16ae03035fc69eaf100ea39657a49baaeef714e25a52575710c34cd48/six-1.16.0-py2.py3-none-any.whl
typing-extensions @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/3d/38/26/2c9b521373bbaf207e658ec81f51aa2a8af7454bfe4d7c15743a6533d5/typing_extensions-3.10.0.0-py3-none-any.whl
urllib3 @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/51/08/ff/7c6aaf6cf9ca83a0bf63c5121f187a40571849e9b17bd8e031898d76c2/urllib3-1.26.5-py2.py3-none-any.whl
wrapt @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/6c/e2/d9/2c022794d212a87320efa16fd1b05654bf6656b6cf0510c072845ecc95/wrapt-1.12.1.tar.gz
yarl @ file:///Users/yoav/Library/Caches/pypoetry/artifacts/5b/3e/b4/b94ca634af6fbb9a677f39777f8a6a78c3d46d9c255399827adbe2a487/yarl-1.6.3-cp39-cp39-macosx_10_14_x86_64.whl
```

**Environment:**
 - Python Version: 3.9
 - OS name and version: darwin (10.15.7)

**Additional context**
It seems that the [builtin handlers](https://github.com/boto/botocore/blob/23ee17f5446c78167ff442302471f9928c3b4b7c/botocore/handlers.py#L942) are not called anymore. I ran git bisect and found that the issue is in this commit 6f077265493310943c09759b49d21993045fe1e7.

What I expect is to get the following error (which is the error I get from botocore):
```
botocore.exceptions.ClientError: An error occurred (InvalidParameterValue) when calling the CopySnapshot operation: The specified keyId key-1234 is invalid
```

What I'm getting:
```
botocore.exceptions.ClientError: An error occurred (InvalidParameterValue) when calling the CopySnapshot operation: Invalid PresignedUrl parameter
sys:1: RuntimeWarning: coroutine 'AioRequestSigner.generate_presigned_url' was never awaited
```

Code to reproduce:

```python
import asyncio

import botocore
import aiobotocore

SNAPSHOT_ID = "snap-012ead0caf39d2e8d"


def botocore_main():
    session = botocore.session.Session()
    ec2 = session.create_client("ec2")
    ec2.copy_snapshot(
        SourceSnapshotId=SNAPSHOT_ID,
        Description=f"Copy of encrypted {SNAPSHOT_ID}",
        Encrypted=True,
        SourceRegion="us-east-1",
        DestinationRegion="us-east-1",
        KmsKeyId="key-1234",
    )


async def main():
    session = aiobotocore.get_session()
    async with session.create_client("ec2") as ec2:
        ret = await ec2.copy_snapshot(
            SourceSnapshotId=SNAPSHOT_ID,
            Description=f"Copy of encrypted {SNAPSHOT_ID}",
            Encrypted=True,
            SourceRegion="us-east-1",
            DestinationRegion="us-east-1",
            KmsKeyId="key-1234",
        )


# botocore_main()
loop = asyncio.get_event_loop()
loop.run_until_complete(main())
```

**Comment by thehesiod** [comment]
will look asap

**Comment by thehesiod** [comment]
i tried repro'ing w/ moto but couldn't, so it seems there's something something that moto isn't doing to trigger it:
Here's the testcase I used:
```python
@pytest.mark.moto
@pytest.mark.asyncio
async def test_ec2_snapshot(ec2_client, topic_arn):
    volume_response = await ec2_client.create_volume(AvailabilityZone="us-east-1", Size=10)
    tag_spec = [
        {"ResourceType": "snapshot", "Tags": [{"Key": "key", "Value": "value"}]}
    ]

    create_snapshot_response = await ec2_client.create_snapshot(
        VolumeId=volume_response["VolumeId"], TagSpecifications=tag_spec
    )

    copy_snapshot_response = await ec2_client.copy_snapshot(
        SourceSnapshotId=create_snapshot_response["SnapshotId"],
        SourceRegion="us-east-1",
        DestinationRegion="us-east-1",
        Encrypted=True,
        TagSpecifications=tag_spec,
        KmsKeyId="key-1234",
    )
```

**Comment by thehesiod** [comment]
I'll need to test against a real snapshot tmrw

**Comment by thehesiod** [comment]
found it, its deep in the bowels :(. will come up with fix

**Comment by thehesiod** [comment]
fixed in 1.3.1, should be out shortly
